MapboxDirections/MBRouteOptions.swift

```diff
@@ -211,6 +211,13 @@ open class RouteOptions: NSObject {
      */
     open var routeShapeResolution = RouteShapeResolution.low
     
+    /**
+     AttributeOptions for the route. Any combination of `.AttributeOptions` can be specified. 
+     
```

Comment:
If it’s the case that `.low` resolution yields the same values (which reflect the `.full` geometry), let’s recommend that the developer set `.full` to make use of these options.